Servers/UI/OJS.Servers.Ui/ClientApp/src/components/administration/contests/participants-in-contest-view/ParticipantsInContestView.tsx

```diff
@@ -45,26 +47,34 @@ const ParticipantsInContestView = (props: IParticipantsInContestView) => {
           open={openCreateModal}
           onClose={() => onModalClose()}
         >
-            <ParticipantForm contestId={contestId} contestName={contestName} />
+            <ParticipantForm
+              contestId={contestId}
+              contestName={contestName}
+              onSuccess={onModalClose}
+              setParentSuccessMessage={setSuccessMessage}
+            />
         </AdministrationModal>
     );
 
     return (
-        <div style={{ marginTop: '2rem' }}>
-            <AdministrationGridView
-              data={data}
-              error={error}
-              filterableGridColumnDef={participantsFilteringColumns}
-              notFilterableGridColumnDef={returnparticipantsNonFilterableColumns(refetch)}
-              queryParams={queryParams}
-              renderActionButtons={renderActions}
-              modals={[
-                  { showModal: openCreateModal, modal: (i) => renderParticipantModal(i) },
-              ]}
-              setQueryParams={setQueryParams}
-              withSearchParams={false}
-            />
-        </div>
+        <>
+            {renderSuccessfullAlert(successMessage)}
+            <div style={{ marginTop: '2rem' }}>
```

Comment:
Cant this inline styling be added to the AdministrationGridView css module?

Reply:
Extracted the style into a separate ParticipantsInContestView.module.scss file.